AttributeError on Python 3

`sys.exc_type` has been deprecated a long time ago and was removed in Python 3 which causes this error:

```py
                response = xmlrpclib.dumps(
>                   xmlrpclib.Fault(1, "%s: %s" % (sys.exc_type.__name__, sys.exc_info()[1])),
                    allow_none=self.allow_none, encoding=self.encoding)
E               AttributeError: module 'sys' has no attribute 'exc_type'

venv3/lib/python3.7/site-packages/kobo/django/xmlrpc/dispatcher.py:112: AttributeError
```

--- a/kobo/django/xmlrpc/dispatcher.py
+++ b/kobo/django/xmlrpc/dispatcher.py
@@ -108,8 +108,10 @@ class DjangoXMLRPCDispatcher(SimpleXMLRPCDispatcher):
                     xmlrpclib.Fault(1, u"%s" % Traceback().get_traceback()),
                     allow_none=self.allow_none, encoding=self.encoding)
             else:
+                exc_info = sys.exc_info()[1]
+                exc_type = exc_info.__class__.__name__
                 response = xmlrpclib.dumps(
-                    xmlrpclib.Fault(1, "%s: %s" % (sys.exc_type.__name__, sys.exc_info()[1])),
+                    xmlrpclib.Fault(1, "%s: %s" % (exc_type, exc_info)),
                     allow_none=self.allow_none, encoding=self.encoding)
 
         return response

